tenseal/cpp/tensors/ckkstensor.cpp

```diff
@@ -1,5 +1,9 @@
 #include "tenseal/cpp/tensors/ckkstensor.h"
 
+#define OP_ADD 0
```

Comment:
It is preferred to use enum classes instead of defines or plain enums, to prevent bugs/automatic casts or conversions.